bin/asinit

```diff
@@ -253,8 +290,8 @@ function ensureGitignore() {
 function ensurePackageJson() {
   console.log("- Making sure that 'package.json' contains the build commands...");
   const entryPath = path.relative(projectDir, entryFile).replace(/\\/g, "/");
-  const buildUntouched = "asc " + entryPath + " -b build/untouched.wasm -t build/untouched.wat --sourceMap --debug";
-  const buildOptimized = "asc " + entryPath + " -b build/optimized.wasm -t build/optimized.wat --sourceMap --optimize";
+  const buildUntouched = "asc " + entryPath + " --target debug";
+  const buildOptimized = "asc " + entryPath + " --target release";
```

Comment:
how about also add `release-fast` and `release-small`? For `release-fast` add `-O3 --noAssert` for `release-small` (-O3z --noAssert)

Reply:
I'm not opposed to this. Can we move this pull request forward with this change as an add-on for later?

Reply:
I'm waiting for help on adding one last "complicated" test.

Reply:
Also favoring to leave it just `release` and `debug` for now and think about presets later. Might even be that we don't need these in asconfig, but can instead add shorthands like `--optimize speed` and `--optimize size` one can use within their targets.

Reply:
`--target debug` and `--target release` seemed to fit like a glove here. My only question is how do we modify "current"  asconfigs when `asinit` is run?

Reply:
For `--target`, yes. Would assume that nobody typically needs more than `release` and `debug` as their targets, and that it makes more sense to add presets to the `--optimize` option or something similar (independent of asconfig) , which is then used within asconfig's targets.